src/App.js

```diff
@@ -20,37 +20,56 @@ What should a cart item look like? 🤔
 console.log(initialStoreItems)
 
 export default function App() {
-  // const [items, setItems] = useState(initialStoreItems)
+ 
   const [cart, setCart] = useState([])
   // Setup state here...
 
   const addToCart = selection => {
```

Comment:
simple and easy method to add and increase item in the cart. nicely planned and executed..